src/services/identityVerificationService.ts

```diff
@@ -76,12 +76,17 @@ export class IdentityVerificationService {
             foreNames.push(clientData.middleName!);
         }
 
+        const documentsChecked = clientData.documentsChecked!;
+        const verificationEvidence = documentsChecked.map((document) => {
+            return { type: document };
+        });
+
         return {
-            // below 3 fields are hardcoded. Need to replace with actual logic in future
+            // below 2 fields are hardcoded. Need to replace with actual logic in future
             acspId: "1234567890",
```

Comment:
I know it's not part of this ticket, but can we assign acspId here if its part of session value now ?

Reply:
Updated